### ISSUE TITLE
About training loss,I used my own dataset whose format is the same as what crowdAI and ,however after several epoches ，The loss value becomes negativeral.can you help me ?Thank you very much！

### ISSUE THREAD

**Issue by zhenghan408** [aatifjiwani/rgb-footprint-extract#11]

**Comment by zhenghan408** [comment]
however ,the loss Keep going down，Finally converged

**Comment by zhenghan408** [comment]
Besides，i think the result is based on semantic segmentation, adding boundary loss.
In the original text:Exponential Weighted Boundary Loss (EWC): Similar to [23], which introduces a third building boundary class,we add a weighted loss function [27] that heavily penalizes wrong predictions close to and on building edges. Our exponential boundary loss generates a weight map w(x) that forces the model to separate neighboring objects into distinct entities

however,in my opinin,Semantic segmentation plus boundary loss is more like instance segmentation。
I don’t know if I have the right idea,

**Comment by aatifjiwani** [comment]
Hey, sorry it took so long to get back to you. If you're using a `beta` that is less than `1.0`, then it is likely the combined loss will converge to be negative. By nature of the F-Beta measure, a `beta` less than 1 will cause the numerator to exceed the denominator so doing `1 - FBeta` will result in a negative number.
